conda/core/link.py

```diff
@@ -926,6 +947,15 @@ def _execute(self, all_action_groups):
                         #   call something that isn't there anymore
                         for axngroup in make_menu_actions:
                             UnlinkLinkTransaction._execute_actions(axngroup)
+
+                # Execute any user-defined post-transaction actions
+                for exc in self.execute_executor.map(
```

Comment:
Probably not necessary to pass to the executor, since there will typically be very few FinalTransactionActions, but if many are implemented this may be useful to have here.